parser-typechecker/src/U/Codebase/Branch/Diff.hs

```diff
@@ -212,6 +223,10 @@ streamNameChanges namePrefix (TreeDiff (DefinitionDiffs {termDiffs, typeDiffs} :
              in (listifyNames name $ adds diff, listifyNames name $ removals diff)
   let nameChanges = NameChanges {termNameAdds, termNameRemovals, typeNameAdds, typeNameRemovals}
   acc <- f namePrefix nameChanges
+  -- acc <-
+  --   if nameChanges == mempty
+  --     then pure mempty
+  --     else f namePrefix nameChanges
```

Comment:
junk?

Reply:
I was toying with that as a performance optimization, it didn't seem to make too much difference for performance, but it does make debugging much easier so I'll enable it after all 👍🏼